pages/common/hledger.md

```diff
@@ -1,20 +1,36 @@
 # hledger
 
-> A plain text accounting software for the command-line.
+> A robust, friendly plain text accounting app (the command-line interface).
 > More information: <https://hledger.org>.
 
-- Add transactions to your journal interactively:
+- Record new transactions interactively, saving to the default journal file:
 
 `hledger add`
 
-- Show the account hierarchy, using a specific journal file:
+- Import new transactions from bank.csv, using bank.csv.rules to convert:
 
-`hledger --file {{path/to/file.journal}} accounts --tree`
+`hledger import {{bank.csv}}`
 
-- Show a monthly income statement:
+- Print all transactions, reading from multiple specified journal files:
 
-`hledger incomestatement --monthly --depth 2`
+`hledger print --file {{2024.journal}} --file {{2023.journal}}`
 
-- Print the amount of cash spent on food:
+- Show all accounts, as a hierarchy, and their types:
 
-`hledger print assets:cash | hledger -f- -I balance expenses:food --depth 2`
+`hledger accounts --tree --types`
+
+- Show asset and liability account balances, including zeros, hierarchically:
+
+`hledger balancesheet --empty --tree --no-elide`
+
+- Show monthly incomes/expenses/totals, largest first, summarised to 2 levels:
+
+`hledger incomestatement --monthly --row-total --average --sort --depth 2`
+
+- Show the assets:bank:checking account's transactions and running balance:
+
+`hledger aregister assets:bank:checking`
+
+- Show the amount spent on food from the assets:cash account:
```

Comment:
```suggestion
- Show the amount spent on food from the `assets:cash` account:
```

Reply:
I wasn't sure if this content is considered markdown; I don't think it is. Eg `tldr` on mac just renders the backquotes as written. In that case, would single quotes or double quotes be preferable ? Backquotes have specific meaning to the shell and would cause problems if a user copies and pastes them.

Reply:
In descriptions, when backticks are used, it is highlighted by our clients in a different style/color which doesn't only mean a command in this context but can also mean something like file names, properties, etc.

i.e. This is from the Git page we have:

![image](https://github.com/tldr-pages/tldr/assets/26346867/a6ef5728-02c0-4c88-82c6-dc2dd170a72e)